content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -424,3 +427,21 @@ namespace.
 
 When each team has their own mesh, cross-mesh communication follows the
 concepts described in the [multiple meshes](#multiple-meshes) model.
+
+## Mesh expansion
```

Comment:
I think we decided not to use the term "Mesh Expansion" because it's not a known industry term and any configuration that adds services to the mesh would be considered "expanding the mesh".
```suggestion
## Virtual machines and bare metal
```